vpcclassicv1/vpc_classic_v1.go

```diff
@@ -1578,7 +1579,7 @@ func (vpcClassic *VpcClassicV1) ListInstanceNetworkInterfaces(listInstanceNetwor
 	if err != nil {
 		return
 	}
-	err = core.UnmarshalModel(rawResponse, "", &result, UnmarshalNetworkInterfaceCollection)
+	err = core.UnmarshalModel(rawResponse, "", &result, UnmarshalNetworkInterfaceUnpaginatedCollection)
```

Comment:
Technically this is an incompatible change.  Since you are still on a "0." version, you just need to bump the minor level to signify a breaking change.